src/components/common/PageHeaderWrapper.tsx

```diff
@@ -64,9 +49,22 @@ export default function PageHeaderWrapper({ children, nextElement }: Props): JSX
     };
   }, [children, nextElement, height]);
 
+  const styles = useMemo<Record<string, any>>(
```

Comment:
Is it necessary to put it inside a useMemo?
Could it be ```
const styles = {
      visibility: scrollDown && sticky ? 'hidden' : 'visible',
      position: sticky ? 'fixed' : undefined,
      top: sticky ? `${TOP_BAR_HEIGHT}px` : undefined,
      width: sticky ? '-webkit-fill-available' : undefined,
      background: sticky ? theme.palette.TwClrBg : undefined,
      boxShadow: sticky ? `0px 3px 3px -3px ${theme.palette.TwClrBaseGray200}` : undefined,
      zIndex: sticky ? 100 : undefined,
    }
```

Reply:
not necessary, i just thought that might cut down the number of times it had to rebuild the map 🤷

Reply:
but it's such a lightweight calc that it probably doesn't matter. I can take it out to simplify